lib/rubocop/cop/salsify/rails_application_record.rb

```diff
@@ -1,11 +1,7 @@
-# encoding: utf-8
-
-require 'rubocop'
-
 module RuboCop
   module Cop
     module Salsify
-      # Check that models subclass ApplicationRecord with Rails 5.0
+      # Check that models subclass ApplicationRecord with Rails 5.0.
```

Comment:
Looks like this cop exists in rubocop now: https://github.com/bbatsov/rubocop/blob/10a7041d23bcd579821b378dd351aeead7c3f082/lib/rubocop/cop/rails/application_record.rb